fun.kikoplay.KikoPlay.yml

```
@@ -241,3 +226,16 @@ modules:
         path: patch/0001-prebuild.patch
       - type: patch
         path: patch/0002-install.patch
+    modules:
+      - name: resources
+        buildsystem: simple
+        build-options:
+          no-debuginfo: true
+        build-commands:
+          - install -Dm644 fun.kikoplay.KikoPlay.metainfo.xml -t $FLATPAK_DEST/share/metainfo
+          - install -Dm644 fun.kikoplay.KikoPlay.png -t $FLATPAK_DEST/share/icons/hicolor/128x128/apps
+        sources:
+          - type: file
+            path: fun.kikoplay.KikoPlay.metainfo.xml
+          - type: file
+            path: fun.kikoplay.KikoPlay.png
```

```suggestion
          - type: file
            path: fun.kikoplay.KikoPlay.metainfo.xml
          - type: file
            path: fun.kikoplay.KikoPlay.png
        post-install:
          - install -Dm644 fun.kikoplay.KikoPlay.metainfo.xml -t $FLATPAK_DEST/share/metainfo
          - install -Dm644 fun.kikoplay.KikoPlay.png -t $FLATPAK_DEST/share/icons/hicolor/128x128/apps
```


this is what I meant

---

sorry.